.github/workflows/proto-buf-publisher.yml

```diff
@@ -0,0 +1,29 @@
+name: Proto Buf Publishing - Action
+# Protobuf runs buf (https://buf.build/) push updated proto files to https://buf.build/cosmwasm/wasmd
+# This workflow is only run when a .proto file has been changed
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'proto/**'
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: bufbuild/buf-setup-action@v0.7.0
+
+      # lint checks
+      - uses: bufbuild/buf-lint-action@v1
+
+      # backward compatibility breaking checks
+      - uses: bufbuild/buf-breaking-action@v1
+        with:
+          against: 'https://github.com/CosmWasm/wasmd.git#branch=master'
```

Comment:
Does this break the build?
This looks useful when we have a v1 branch. Currently this project is not stable and we may have to do breaking changes.

Reply:
It will not fail the build, just adds comments to let you know that those are breaking changes.